packages/core/package.json

```diff
@@ -48,6 +48,10 @@
       "types": "./jwt.d.ts",
       "import": "./jwt.js"
     },
+    "./common": {
```

Comment:
Let's not add another submodule, just use `@auth/core/types`

Reply:
not types, they are utils functions that help you create client functions